### ISSUE TITLE
adopt summary writer

### ISSUE THREAD

**Issue by c-schicho** [c-schicho/ZeroInitialization#10]
log the validation accuracy, validation loss.
adopt the steps to updates instead of processed examples (x-axis).
